src/Decred.BlockExplorer/AddressRepository.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using Dapper;
+
+namespace Decred.BlockExplorer
+{
+    public interface IAddressRepository
+    {
+        /// <summary>
+        /// Determines the unspent balance of each address at a point in time.
+        /// </summary>
+        /// <param name="maxBlockHeight"></param>
```

Comment:
You add xml comments for params/return, but did not provide comments itself. Either remove xml comments, or add descriptions.